java/kafka/consumer/src/main/java/KafkaConsumerExample.java

```diff
@@ -27,48 +22,68 @@
 public class KafkaConsumerExample {
     private static final Logger log = LogManager.getLogger(KafkaConsumerExample.class);
 
+    public enum TracingSystem {
+        OPENTRACING,
+        OPENTELEMETRY;
+
+        public static KafkaConsumerExample.TracingSystem forValue(String value) {
+            switch (value) {
+                case "opentracing":
```

Comment:
we are actually using "jaeger" across all the Strimzi projects because it was the implementation of OpenTracing, so I would use that.